### DISCUSSION TITLE
TP4 - ejercicio 9 - Elimina duplicados

### DISCUSSION THREAD

**Discussion by agusscarava** [INGCOM-UNRN-P1/cursada-2023#150] [Preguntas y respuestas] [answered]
Hola, tengo un tema para el ejercicio de eliminar duplicados, y es que en la función no entra nunca al bucle que mueve los dígitos. No logro detectar porque no está funcionando. El programa Compila y ejecuta, pero imprime el arreglo tal cual entra.
Esta es la función: 

https://github.com/INGCOM-UNRN-P1/tp4-2023-agusscarava/blob/main/ejercicio9.c

**Comment by martinvilu** [comment, marked as answer]
¿Por que lo marcaste como cerrado? Mañana lo reviso en el repositorio.

**Reply by agusscarava** [reply]
Lo marque como cerrado porque voli a armar la lógica con ayuda de otro compañero y quedo andando. No me dejaba marcarlo como respondido

**Reply by martinvilu** [reply]
Fantastico!
La próxima, mejor enchufa un comentario y marcalo como la respuesta

**Comment by martinvilu** [comment]
Con el `largo` en cero, la función no va a ir a ningún lado.

Separá la entrada de la salida siendo que no estás usando el retorno de la función.

**Reply by agusscarava** [reply]
Okei okei, lo voy a tener en cuenta, gracias Martin

**Comment by agusscarava** [comment]
[image: images.jpeg]
aaa okey no la tenía esa
